tasties_app/views.py

```diff
@@ -1,9 +1,11 @@
 from django.shortcuts import render
+from tasties_app.models import Category
 
 
 def index(request):
     return render(request, 'tasties_app/index.html',)
 
 
 def base(request):
-    return render(request, 'tasties_app/base.html',)
+    categories_list = Category.objects.all()
+    return render(request, 'tasties_app/base.html', {'objects': categories_list})
```

Comment:
Please change the name of the objects as well.
You will also need to change it in the HTML file.